plotly/tests/test_core/test_tools/test_make_subplots.py

```diff
@@ -103,98 +103,99 @@ def test_single_plot(self):
         expected = Figure(
             data=Data(),
             layout=Layout(
-                xaxis1=XAxis(
+                xaxis=XAxis(
```

Comment:
I think these changes from `xaxis1` -> `xaxis` in the constructors are no longer neccesary after the fix in the following commit:

*Rework handling of unknown kwargs in datatype classes* (a89dcb57f5c0f966ccc990d6c1f6dc5687430fba)
@Kully

Reply:
Doesn't look like it's working:
```
expected = Figure(
    data=Data(),
    layout=Layout(
        xaxis1=XAxis(
            domain=[0.0, 1.0],
            anchor='y1'
        ),
        yaxis1=YAxis(
            domain=[0.0, 1.0],
            anchor='x1'
        )
    )
)
expected.to_plotly_json()
```
Output:
```
{'data': [], 'layout': {}}
```